CONTRIBUTING.md

```diff
@@ -17,35 +17,17 @@ some of the factors we'll consider when reviewing your code.
 
 ## Development branches
 
-Please create a fork of the LORIS repository on your own GitHub account and
-push changes to local branches rather than pushing new branches directly 
-to our repository.
-
-You should base your pull requests on one of the following branches
-depending on the kind of change you are making:
-
-#### Bug Fixes
-   - Branch: `bugfix` 
-   - Content: Generally these changes do not require SQL scripts
-   and are concise with the sole objective to correct a single problem
-   in the code.
-
-#### Minor Changes and Small Features
-   - Branch: `minor` 
-   - Content: Features affecting self-contained components such
-   as modules. Additions to Libraries, API, or modules that do not change
-   any function signatures.
-
-#### Major Changes, Non Backwards-Compatible Changes and Large Features
-   - Branch: `major` 
-   - Content: Any change modifying a function signature in a
-   library class. Features require extensive LORIS-wide testing. New
-   complex systems and features spanning across multiple modules and
-   libraries. Deprecated functions clean-up.
-
-For more information about making well-organized pull requests,
-please read our in-depth Wiki page, 
-["Contributing to the Code"](https://github.com/aces/Loris/wiki/Contributing-to-the-Code).
+For the most part, changes to the codebase should be sent to the `master`
+branch, which is the default.
+
+Small bug fixes for a given version of LORIS should be sent to the branch named
+for that version. For example, fixing a bug in LORIS version 22 should be sent
+to the `22.0-release` branch.
+
+When fixing a bug present in multiple versions of LORIS, send your fix to the
```

Comment:
This paragraph shouldn't be here, there should be a statement about only the latest version of LORIS being supported in its place. Bug fixes for older releases are only done in exceptional circumstances where a production instance can't be upgraded because of a bug.